src/main/java/com/jcabi/log/VerboseProcess.java

```diff
@@ -107,6 +107,16 @@
     private transient boolean closed;
 
     /**
+     * Maximum number of log lines for a stack trace
+     */
+    public static final int DEFUALT_MAX_LENGTH = 1000;
+
+    /**
+     * Maximum number of log lines for a stack trace
+     */
+    public static volatile int maxStackLength = DEFUALT_MAX_LENGTH;
```

Comment:
@dean-e-clark can we somehow avoid static mutable data that is visible for everyone?